nshimiyejayd/my-brand
# #5r3h2E8L Implementing About page

#### What does this PR do?
This PR implements About page

#### Description of Task to be completed?
-Implement About page

#### How should this be manually tested?
- Clone the [repo](https://github.com/nshimiyejayd/my-brand)
-  Checkout branch `feat-implement-about-page-5r3h2E8L`
- Open about.html in your browser to check About page

#### What are the relevant pivotal tracker stories?
[#5r3h2E8L](https://trello.com/c/5r3h2E8L/19-about-page)
